CMakeLists.txt

```diff
@@ -72,7 +72,7 @@ ExternalProject_Add(lame
 ExternalProject_Get_Property(lame BINARY_DIR)
 ExternalProject_Get_Property(lame SOURCE_DIR)
 
-option(PYTHON_VERSIONS "Python versions to build" "3.8;3.9;3.10;3.11;3.12")
+option(PYTHON_VERSIONS "Python versions to build" "3.8;3.9;3.10;3.11;3.12;3.13")
 # New CMake doesn't take the default above for some reason...
 if (${PYTHON_VERSIONS} STREQUAL "OFF")
     set(PYTHON_VERSIONS "3.7;3.8;3.9;3.10;3.11")
```

Comment:
Also missing here, which apparently was missed for 12 as well...

Reply:
Always the way.  Making small mistakes is often the way.  Its hard to keep everything updated every time someone somewhere updates something.  Still checks look like they are going well.